source/robot_model/README.md

```diff
@@ -24,23 +24,23 @@ At creation, it uses the parser from `pinocchio` to read the `URDF` file and fil
 
 ## Robot kinematics
 
-Most common robotics functionalities are implemented such `forward_geometry`, `inverse_geometry` (in development),
-`forward_kinematic`, `inverse_kinematic`, ...
+Most common robotics functionalities are implemented such `forward_kinematics`, `inverse_kinematics`,
+`forward_velocity, `inverse_velocity`, ...
 
 ```cpp
-// forward geometry: pose of a robot frame from the robot joint positions
+// forward kinematics: pose of a robot frame from the robot joint positions
 state_representation::JointPositions jp = state_representation::JointPositions::Random("myrobot", 7);
-state_representation::CartesianPose pose = model.forward_geometry(jp);
-// inverse geometry: joint positions from a pose
+state_representation::CartesianPose pose = model.forward_kinematics(jp);
+// inverse kinematics: joint positions from a pose
 state_representation::CartesianPose cp = state_representation::CartesianPose::Random("eef");
-state_representation::JointPositions jp = model.forward_geometry(cp);
-// forward kinematic: twist of a robot frame from the robot joint velocities and positions
+state_representation::JointPositions jp = model.inverse_kinematics(cp);
+// forward velocity kinematic: twist of a robot frame from the robot joint velocities and positions
```

Comment:
is there an `s` missing in `kinematic` ?

Reply:
yes indeed